rollouts/api/v1alpha1/rollout_types.go

```diff
@@ -84,12 +85,17 @@ type SecretReference struct {
 	Name string `json:"name,omitempty"`
 }
 
-// +kubebuilder:validation:Enum=CEL
+// +kubebuilder:validation:Enum=allClusters;custom
```

Comment:
Any thoughts using PascalCase here? Similar to Strategy (AllAtOnce, RollingUpdate, Progressive).

Reply:
Good catch. Fixed it.